ipfs/src/error.rs

```diff
@@ -95,3 +95,11 @@ impl From<ServerError> for String {
 		}
 	}
 }
+
+impl ::std::fmt::Display for ServerError {
+	fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
+
```

Comment:
Nitpick, Display and Debug provide the same imp.

More intuitive to have different impls IMO.